src/node-backend/routes/users.js

```diff
@@ -0,0 +1,52 @@
+const express = require("express");
+const router = express.Router();
+const pool = require("../maria-helper/database");
+// const bcrypt = require("bcrypt");
+
+/*
+to test on postman:
+url: localhost:3000/user/getUser
+body: {"id": <user id>} (setup on HeidiSQL)
+*/
+router.post("/getUser", async function (req, res) {
+  try {
```

Comment:
I would do the query in a file called db-adapter or something like that to be more modular :)